dev/tests/hu.elte.txtuml.xtxtuml.tests/src/hu/elte/txtuml/xtxtuml/tests/parser/XtxtUMLParserTestUtils.xtend

```diff
@@ -136,7 +136,7 @@ class XtxtUMLParserTestUtils {
 		assertTrue(member instanceof TUAttribute);
 		val attribute = member as TUAttribute;
 
-		assertEquals(visibility, attribute.prefix.visibility);
+		assertEquals(visibility, attribute.prefix.modifiers.visibility);
```

Comment:
Yes, the tests were skipped intentionally. If you write them, I will happily include them in this pull request.